### ISSUE TITLE
How To Support Flash

### ISSUE THREAD

**Issue by dyjae** [goxr3plus/JavaFX-Web-Browser#16]
I want to open flash web , but can't load flash plugin

**Comment by goxr3plus** [comment]
Yes you are right, i don't really know yet how support plug-ins because JavaFX browser is based on Safari Browser Engine if i am not wrong :).

I need to make a new release for this Browser for Java 11 :).
